.github/workflows/flatpak.yml

```diff
@@ -34,10 +34,21 @@ jobs:
       with:
         submodules: 'recursive'
 
+    - name: Install packages
+      if: success() && (github.event_name != 'pull_request' || env.SEEKING_TESTERS == '1')
+      run: dnf install -y jq
```

Comment:
I added this as a separated step for two reason:
- Make the job look cleaner, `dnf` added many logs
- Prepare for docker if aarch64 is added (eg https://github.com/tytan652/obs-studio/pull/2/files)